docs/support/troubleshooting.md

```diff
@@ -5,6 +5,7 @@
 This error happens when the commit associated with the last release cannot be found in the branch history.
 
 Multiple situation can cause this issue:
+
```

Comment:
Sorry to nitpick, but can you also remove this additional new line as it's not related to your addition?